test/emqtt_SUITE.erl

```diff
@@ -446,6 +447,12 @@ t_connected(Config) ->
     Clientid = gen_statem:call(C, clientid),
     [ ?assertMatch(Clientid, Value) || {Key, Value} <- emqtt:info(C), Key =:= clientid].
 
+t_low_mem_opts(Config) ->
+    Port = ?config(port, Config),
+    {ok, C} = emqtt:start_link([{port, Port}, {low_mem, true}]),
+    %% TODO: check min_heap_size, min_bin_vheap_size
```

Comment:
For some reason, those values are not set to `16`.  Maybe it overrides to an internal minimum value.

```
Failure/Error: ?assertEqual([], process_info ( C , [ min_heap_size , min_bin_vheap_size , fullsweep_after ] ))
  expected: []
       got: [{min_heap_size,233},
             {min_bin_vheap_size,46422},
             {fullsweep_after,1000}]
      line: 455
```